packages/genesys-spark-components/src/components/beta/gux-form/gux-form-footer/gux-form-footer.tsx

```diff
@@ -0,0 +1,36 @@
+import { Component, h, JSX, Prop, Element } from '@stencil/core';
+import { GuxFormFooterPlacement } from './gux-form-footer.types';
+import { trackComponent } from '@utils/tracking/usage';
+
+/**
+ * @slot - Slot for elements.
+ */
+
+@Component({
+  styleUrl: 'gux-form-footer.scss',
+  tag: 'gux-form-footer',
+  shadow: true
+})
+export class GuxFormFooter {
+  @Element()
+  root: HTMLElement;
+
+  @Prop()
+  placement: GuxFormFooterPlacement = 'page-desktop';
+
+  componentWillLoad(): void {
+    trackComponent(this.root, { variant: this.placement });
+  }
+
+  render(): JSX.Element {
+    return (
+      <footer
+        class={{
```

Comment:
Open to opinions on this should we have the footer element in the light DOM or have it shadowDOMed. I left it here for now as I had all the stylings setup and wanted to make sure. I am not sure if devs can do anything directly with the footer element so encapsulating might be fine ?

Reply:
As at the moment the `header` element is in the light dom I think we should do the same with the `footer` element.

Reply:
By this do you mean to get rid of the `gux-form-footer` completely or just remove the `footer` from the shadowDOM and slot it instead ?

Reply:
I removed it from the shadowDOM so now it can just be slotted to the `gux-form-footer`

Reply:
The `gux-form-footer` will have to stay though I think due to there being different placements.